src/precompile.jl

```diff
@@ -0,0 +1,196 @@
+precompile(Tuple{Type{Union{}}})
```

Comment:
Is `precompile` still the way to do this? a lot of packages are just running the function. At least you know it it breaks that way.

Reply:
SnoopPrecompile's `@precompile_all_calls` might be a good idea https://timholy.github.io/SnoopCompile.jl/dev/snoop_pc/

Reply:
Well, running is also fine, but you can only do that on smaller packages geojsons, and the test geojson we package didn't actually decrease the runtime much if at all for our 24MB usecase. So I think this is the best. It's also easy to make changes to this code later on or even remove it.

The tests will be responsible to detect breakage.

Reply:
I actually ran with @precompile_all_calls first, but that didn't seem to do much either. Might be my setup.